src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -463,10 +463,12 @@ namespace winrt::TerminalApp::implementation
                         // 1 is important to make sure that the effects of
                         // something like `colortool` are cleared when setting
                         // the scheme.
+                        /*

```

Comment:
Maybe I'm not understanding how the inheritable settings are supposed to work. The issue I was seeing is that when later trying to get the `AppliedColorScheme` in Pane, it would always return the one originally set when the pane was created, or if it was changed just null. Is there maybe something wrong with my declaration in `TerminalSettings.h`?

Reply:
Figured it out, didn't realize that I needed to explicitly `Clear` the setting compared to setting it to a falsey value.